package.json

```
@@ -8,6 +8,9 @@
     "import": "./dist/esm/index.js",
     "default": "./dist/esm/index.js"
   },
+  "files": [
+    "dist/*"
```

I think it would be enough to specify either just the dir name or the dir name suffixed with forward slash. try with npm pack locally perhaps before committing the change 

```suggestion
    "dist"
```